pkg/epp/common/config/loader/configloader.go

```diff
@@ -135,18 +135,18 @@ func validateConfiguration(theConfig *configapi.EndpointPickerConfig) error {
 	names := make(map[string]struct{})
 
 	for _, pluginConfig := range theConfig.Plugins {
-		if pluginConfig.PluginName == "" {
-			return errors.New("plugin reference definition missing a plugin name")
+		if pluginConfig.Type == "" {
+			return errors.New("plugin definition missing a plugin type")
```

Comment:
nit: which plugin is missing a type?
adding this info might help fixing the issue when it happens